src/docs/timestamp-global.dox

```diff
@@ -110,31 +107,17 @@ be useful for backup scenarios where rolling back to a stable timestamp
 isn't possible and it's useful for a checkpoint to contain the most recent
 possible data.
 
-@subsection timestamp_global_forcing Forcing global timestamps
-
-\warning
-The WT_CONNECTION::set_timestamp method includes a \c force flag which allows
-applications to violate the usual constraints.  For example, this can be used to
-move \c stable_timestamp backward.  Violating these constraints can lead to
-undefined or incorrect behavior.  For example, while moving \c oldest_timestamp
-backward with this mechanism may make some historical data accessible again,
-other obsolete historical data may have already been discarded, and no
-guarantees of consistency are made.
```

Comment:
Did you take this out because it's undocumented? It seems useful to tell users that can hurt themselves with this (even if it's undocumented).

Reply:
Yes, I took it out because it's undocumented, and my understanding is there's only a single user left in MDB and it's going away.